net.lr.example.kubernetes/src/main/java/net/lr/ds/hello/Index.java

```diff
@@ -17,16 +18,76 @@
 public class Index extends HttpServlet implements Servlet {
 
 	private static final long serialVersionUID = 1L;
+	private static final UUID ID = UUID.randomUUID();
 
 	@Override
 	protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
 		PrintWriter wr = resp.getWriter();
-		wr.write("<html><body>\n");
-		wr.write("<p><a href=\"/control/notready\">Report not ready for 60 seconds</a></p>");
-		wr.write("<p><a href=\"/control/notalive\">Report not alive</a></p>");
-		wr.write("<p><a href=\"/systemready\">Show ready state</a></p>");
-		wr.write("<p><a href=\"/systemalive\">Show alive state</a></p>");
 
+
+		wr.write("<html>\n<head>\n" +
```

Comment:
Not sure if the better styling is worth the overhead. It is much more difficult to understand this now.

Reply:
Well... it was motivated by two factors: 
* consistent styles between the buttons and links
* making it prettier

Reply:
My main motivation was to keep the code as simple as possible. So someone looking at it instantly understands what it does. Maybe we can use a stylesheet to separate the code for triggering the actions from the styling. I will take a look at it.

Reply:
I think the best thing is if you apply the styling like you proposed and we work from there.